### DISCUSSION TITLE
Refund Request

### DISCUSSION THREAD

**Discussion by Kr-LeeSangMin** [twbs/bootstrap#41028] [Q&A]
Hello,

I am from South Korea, and I am writing this message through translation, so I apologize if anything sounds awkward.

I previously requested a refund for a Bootstrap theme via email, but I haven’t received any response.

Is there another way I can proceed with this request, or is there no alternative?

Thank you for your understanding.

**Comment by lenantiri** [comment]
wrong message buddy ...

**Comment by julien-deramond** [comment]
Hello @Kr-LeeSangMin 
Unfortunately, we can't assist with this issue in this repository, as Bootstrap Themes operates independently.
You could try submitting another request. Alternatively, @mdo might be able to provide some guidance.

**Comment by Kr-LeeSangMin** [comment]
Thank you for your reply.

It seems like the only way to get in touch is through email, but they don’t seem to read it even though I’ve sent multiple messages.
I think I might have to give up.
If you happen to come across this post, please take it as a reference to avoid having the same experience as me.

**Comment by khandelwal-aman** [comment]
@Kr-LeeSangMin did you received any response from anyone on the issue?

**Reply by Kr-LeeSangMin** [reply]
Thank you for your interest.
I still haven't resolved the issue.
The person in charge still isn't checking their emails.
